test/OrchardCore.Tests/Apis/GraphQL/Queries/RecentBlogPostsQueryTests.cs

```diff
@@ -35,11 +35,17 @@ public async Task ShouldListBlogPostWhenCallingAQuery()
                         .WithField("displayText");
                 });
 
-            var nodes = result["data"]["recentBlogPosts"];
+            var jsonArray = result["data"]?["recentBlogPosts"]?.AsArray();
 
-            Assert.Equal(2, nodes.AsArray().Count);
-            Assert.Equal("Some sorta blogpost in a Query!", nodes[0]["displayText"].ToString());
-            Assert.Equal("Man must explore, and this is exploration at its greatest", nodes[1]["displayText"].ToString());
+            Assert.NotNull(jsonArray);
+            Assert.Equal(2, jsonArray.Count);
+
+            // The RecentBlogPosts query sorts the content items by CreatedUtc. If the
+            // test is executing too fast, both blog entries may have the same CreatedUtc
+            // value and ordering becomes random. Because of this, we do not assert the order
+            // of the result.
+            Assert.Contains("Some sorta blogpost in a Query!", jsonArray.Select(node => node["displayText"]?.ToString()));
```

Comment:
Can you save the result of `jsonArray.Select(node => node["displayText"]?.ToString())` into a variable and use it in both lines?

Reply:
Sure, done!